pkg/bpf/bpf_kmesh_common.go

```diff
@@ -178,17 +262,25 @@ func (sc *BpfSockConn) Attach() error {
 }
 
 func (sc *BpfSockConn) Detach() error {
-	var value reflect.Value
-
 	if err := sc.close(); err != nil {
 		return err
 	}
 
-	value = reflect.ValueOf(sc.KmeshCgroupSockObjects.KmeshCgroupSockPrograms)
+	var value reflect.Value
+	if sc.Info.KernVerLt5_13 {
```

Comment:
instead of juding KernVerLt5_13 everywhere, can we abstract a interface, providing 
```
GetXXXProg
GetXXXMap
```